Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG APP_INSIGHTS_AGENT_VERSION=3.2.4
+ARG APP_INSIGHTS_AGENT_VERSION=3.4.8
```

Comment:
@shivajihmcts can you please upgrade to ARG APP_INSIGHTS_AGENT_VERSION=**3.4.9**